src/arch/xtensa/CMakeLists.txt

```diff
@@ -602,7 +602,7 @@ else()
 		bin ALL
 		COMMAND ${CMAKE_COMMAND} -E copy sof-${fw_name}.ri ${PROJECT_BINARY_DIR}/sof-${fw_output_name}.ri
 		COMMAND ${CMAKE_COMMAND} -E copy sof-${fw_name}.ri ${PROJECT_BINARY_DIR}/sof.ri
-		DEPENDS run_meu bin_extras glue_binary_files sof_dump reproducible.ri
+		DEPENDS run_meu bin_extras glue_binary_files sof_dump #reproducible.ri
```

Comment:
oh, this looks like it's sneaked in from another patch ?

Reply:
yes, sorry :)
Removed